src/Stratis.Bitcoin.Features.SmartContracts.Tests/SmartContractTransactionServiceTests.cs

```diff
@@ -977,6 +977,14 @@ public void BuildFeeEstimationContext_Recipient_Is_Not_P2PKH()
         }
         */
 
+        [Fact]
```

Comment:
Could we have more tests that demonstrate what this actually does?

Reply:
Added.